# Goal
p_goal_x: 15
p_goal_y: 0
p_goal_z: 0


# A*
max_vel: 2.0
max_acc: 3.0

use_height_limit: true
height_limit_max: 1.8
height_limit_min: 0.0
sample_z_acc: false

a_star_acc_sample_step: 1.0
a_star_search_time_step: 0.6

risk_threshold_motion_primitive: 0.3
expand_safety_distance: 0.2

# Corridor
risk_threshold_single_voxel: 0.15
risk_threshold_corridor: 3.0


# Trajectory planner
planning_time_step: 0.05
trajectory_piece_max_size: 8

pos_factor: 0.0
vel_factor: 0.0
acc_factor: 0.0
jerk_factor: 0.0
snap_factor: 1.0  # minimum snap
delta_corridor: 0.4

max_vel_optimization: 2.5 # Constraint in optimization. A little larger than max_vel
max_acc_optimization: 3.5 # Constraint in optimization. A little larger than max_acc


# Mapping
observation_stddev: 0.1
localization_stddev: 0 

# Visualization
rviz_map_center_locked: false

# Others
max_differentiated_current_a: 3.0

# Goal
goal_reach_threshold: 2.0


